packages/backend/src/peripherals/database/TotalSupplyRepository.ts

```diff
@@ -0,0 +1,82 @@
+import { Logger } from '@l2beat/shared'
+import { AssetId, ChainId, UnixTime } from '@l2beat/shared-pure'
+import { TotalSupplyRow } from 'knex/types/tables'
+
+import { BaseRepository, CheckConvention } from './shared/BaseRepository'
+import { Database } from './shared/Database'
+
+export interface TotalSupplyRecord {
+  timestamp: UnixTime
+  totalSupply: bigint
+  assetId: AssetId
+  chainId: ChainId
+}
+
+export interface DataBoundary {
```

Comment:
Copy-pasta, this is not used anywhere

Reply:
https://github.com/l2beat/l2beat/pull/1663/commits/f38d327505143c45f44435ca720343899f7683a5